config/bundles.php

```diff
@@ -0,0 +1,52 @@
+    Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle::class => ['all' => true],
+    FOS\RestBundle\FOSRestBundle::class => ['all' => true],
+    JMS\SerializerBundle\JMSSerializerBundle::class => ['all' => true],
+    Bazinga\Bundle\HateoasBundle\BazingaHateoasBundle::class => ['all' => true],
+    Massive\Bundle\SearchBundle\MassiveSearchBundle::class => ['all' => true],
+    FOS\HttpCacheBundle\FOSHttpCacheBundle::class => ['all' => true],
+    Sulu\Bundle\SearchBundle\SuluSearchBundle::class => ['all' => true],
+    Sulu\Bundle\PersistenceBundle\SuluPersistenceBundle::class => ['all' => true],
+    Sulu\Bundle\ContactBundle\SuluContactBundle::class => ['all' => true],
+    Sulu\Bundle\MediaBundle\SuluMediaBundle::class => ['all' => true],
+    Sulu\Bundle\SecurityBundle\SuluSecurityBundle::class => ['all' => true],
+    Sulu\Bundle\CategoryBundle\SuluCategoryBundle::class => ['all' => true],
+    Sulu\Bundle\SnippetBundle\SuluSnippetBundle::class => ['all' => true],
+    Sulu\Bundle\ContentBundle\SuluContentBundle::class => ['all' => true],
+    Sulu\Bundle\TagBundle\SuluTagBundle::class => ['all' => true],
+    Sulu\Bundle\WebsiteBundle\SuluWebsiteBundle::class => ['all' => true],
+    Sulu\Bundle\LocationBundle\SuluLocationBundle::class => ['all' => true],
+    Sulu\Bundle\HttpCacheBundle\SuluHttpCacheBundle::class => ['all' => true],
+    Sulu\Bundle\WebsocketBundle\SuluWebsocketBundle::class => ['all' => true],
+    Sulu\Bundle\DocumentManagerBundle\SuluDocumentManagerBundle::class => ['all' => true],
+    Sulu\Bundle\HashBundle\SuluHashBundle::class => ['all' => true],
+    Sulu\Bundle\CustomUrlBundle\SuluCustomUrlBundle::class => ['all' => true],
+    Sulu\Bundle\RouteBundle\SuluRouteBundle::class => ['all' => true],
+    Sulu\Bundle\MarkupBundle\SuluMarkupBundle::class => ['all' => true],
+    DTL\Bundle\PhpcrMigrations\PhpcrMigrationsBundle::class => ['all' => true],
+    Massive\Bundle\BuildBundle\MassiveBuildBundle::class => ['all' => true],
+    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
+    Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle::class => ['dev' => true, 'test' => true],
+    Symfony\Bundle\WebServerBundle\WebServerBundle::class => ['dev' => true, 'test' => true],
+    Sulu\Bundle\TestBundle\SuluTestBundle::class => ['dev' => true, 'test' => true],
+    Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true, 'test' => true],
+    // Admin
+    Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true, 'admin' => true],
+    Sulu\Bundle\AdminBundle\SuluAdminBundle::class => ['all' => true,'admin' => true],
+    Sulu\Bundle\CollaborationBundle\SuluCollaborationBundle::class => ['all' => true,'admin' => true],
+    Sulu\Bundle\PreviewBundle\SuluPreviewBundle::class => ['all' => true,'admin' => true],
+    // Website
+    Symfony\Cmf\Bundle\RoutingBundle\CmfRoutingBundle::class => ['all' => true,'website' => true],
```

Comment:
The above 5 lines miss a space before the second array entry. And apart from that, that would make problems if we introduce a `website` or `admin` environment, right?

Reply:
@danrot yes it would block but we did discuss this that this this is aslong as we have 2 kernel setup the only solution if we want to support flex.